classes/class.cptui_admin_ui.php

```diff
@@ -93,7 +93,7 @@ public function get_td_end() {
 	 * @param array $args Array of arguments.
 	 * @return string $value Opening `<fieldset>` tag.
 	 */
-	public function get_fieldset_start( $args = [] ) {
```

Comment:
Short array syntax is going to be one that I put my foot down on for not reverting as we already dedicated time to convert them to the shorthand which is what I want to use going forward.